src/treemodel/selection.js

```diff
@@ -229,44 +229,49 @@ export default class Selection {
 	/**
 	 * Sets collapsed selection in the specified location.
 	 *
-	 * The location can be specified as:
+	 * The location can be specified in the same form as {@link engine.treeModel.Position.createAt} parameters.
 	 *
-	 * * a {@link engine.treeModel.Position position},
-	 * * parent element and offset (offset defaults to `0`),
-	 * * parent element and `'END'` (sets selection at the end of that element),
-	 * * node and `'BEFORE'` or `'AFTER'` (sets selection before or after the given node).
-	 *
-	 * @fires {@link engine.treeModel.Selection#change:range change:range}
+	 * @fires engine.treeModel.Selection#change:range
 	 * @param {engine.treeModel.Node|engine.treeModel.Position} nodeOrPosition
 	 * @param {Number|'END'|'BEFORE'|'AFTER'} [offset=0] Offset or one of the flags. Used only when
 	 * first parameter is a node.
 	 */
 	collapse( nodeOrPosition, offset ) {
```

Comment:
There's [collapse()](https://developer.mozilla.org/en-US/docs/Web/API/Selection/collapse) method in the DOM. Besides, that method wasn't added in this PR.